### PR TITLE
Lunarium missions - grammar fixes

### PR DESCRIPTION
**Pull request by opusforlife2**
**Content (Artwork / Missions / Jobs)**

### Summary

Some grammar fixes, both for regular English, and for Coalition grammar consistency.

### CODE REVIEW

**Review by OcelotWalrus — APPROVED** [review]
Looking good for me.

File: data/coalition/lunarium intro.txt [hunk 1]
```diff
@@ -1285,20 +1285,20 @@ mission "Lunarium: Evacuation 6"
...
+			`	He thinks for a few minutes, oftentimes opening up his mouth as if deciding to speak, but ultimately going back to reflecting on what to say. "When decided to... desert, I had, work I did on covering up my escape of the facility where stationed, I was. After forcing an... an incident, meant to erase all traces of mine, fled and hid in civilian ships for a while, I had, until found by the Lunarium I was." He smirks lightly in self-deprecation. "Thought I had that aware enough of their patterns and methods, I was, enough to locate them. Find me first, they did though. Taken in, I was, and helping them since, I have been."`
```
Comment (Arachi-Lover) [inline data/coalition/lunarium intro.txt:1301]: Why change bitterness to self-deprecation?

File: data/coalition/lunarium intro.txt [hunk 2]
```diff
@@ -1619,30 +1619,30 @@ mission "Lunarium: Combat Training 1"
...
 					goto help
 				`	"Your friends only told me what I was really doing after I was done. I don't want to help you any further."`
-			`	"Why, interested are you in helping the Heliarchs instead? Worked with them, some of our members had. An arbiter, Elirom, one of them, was. One of the highest ranks, that is, second only to the consuls themselves," she responds, getting closer to you. "Change, for the better, the Coalition soon will. Difficult it will be, to take down the Heliarchy. Very strong they are, so greatly appreciated any help would be, and appropriately reward those who helped us, we will, once free from them we are."`
+			`	"Why, interested are you in helping the Heliarchs instead? Worked with them, some of our members have. An arbiter, Elirom, one of them, was. One of the highest ranks, that is, second only to the consuls themselves," she responds, getting closer to you. "Change for the better, the Coalition soon will. Difficult it will be, to take down the Heliarchy. Very strong it is, so greatly appreciated any help would be. Of course, once free from them we are, appropriately reward, we will, help us, those who did."`
```
Comment (Arachi-Lover) [inline data/coalition/lunarium intro.txt:1632]: That end reads very odd, with the many commas pausing over and over.

File: data/coalition/lunarium intro.txt [hunk 3]
```diff
@@ -1735,7 +1735,7 @@ mission "Lunarium: Combat Training 2"
...
 		conversation
-			`Pyakri contacts you right as you land, by a text message in your monitor. "A great help you were, Captain! More... veteran members, if you could call them that, the crew I brought are, so much trouble with their morale, I had not, but still worried, the pirates had me. Used to human ships, we are not, so fly them well enough to combat such fleets, we cannot. Not yet. Now that guided us through this first fight, you have, work on our own for a while - taking smaller, simpler bounties - we will. Contact you again, I shall, once done with our training, we are."`
+			`Pyakri contacts you right as you land, by a text message in your monitor. "A great help you were, Captain! More... inveterate members, if call them that you could, the crew with me are, so have much trouble with their morale, I did not, but still have me worried, the pirates did. Used to human ships, we are not, so fly them well enough to combat such fleets, we cannot. Not yet. Now that guided us through this first fight, you have, work on our own for a while - taking smaller, simpler bounties - we will. Contact you again, I shall, once done with our training, we are."`
```
Comment (Arachi-Lover) [inline data/coalition/lunarium intro.txt:1738]: Why the change from veteran to inveterate? Idk if it's just me but I had to google to find out what inveterate means, and even then I don't really see it as having the same intended meaning.

File: data/coalition/lunarium intro.txt [hunk 4]
```diff
@@ -2012,11 +2012,11 @@ mission "Lunarium: Join"
...
+			`	"Only prove the Heliarchs right, that last question would," Elirom chimes in. "Matter not, it does, if a thousand times the footage disproves their narrative. If even one thing, pick and spin as proof of their side they can, fruitless, the effort will be." He pauses, then hits the wall again, with less force. "Play it again, could you?" Oobat obliges, and plays the recording once again. Elirom has it paused a few times over the course of the video, breaking out a device to write down some notes. Once he's finished, he has the last portion repeated a few times still, seeming particularly interested in the Quarg's changing pigment. "Onyx black..." he ponders out loud, "what was, again, the emotion of that color?" Snapping out of the trance only after a few more repetitions, he hands Oobat the notes. "Right they were about the Quarg wanting to take the rings back... A shame it is, that use that point against them we cannot. Though, maybe, if interested in the rings they still are after so long..."`
```
Comment (Arachi-Lover) [inline data/coalition/lunarium intro.txt:2018]: Was removing the "it" from the "the emotion of that color, what was it again?" bit intentional? The new version reads a bit weird without it.

File: data/coalition/lunarium intro.txt (hunk 1 above)
Comment (Arachi-Lover) [reply data/coalition/lunarium intro.txt:1301]: Does the placement there necessarily mean the bitterness is about that? The idea I had for this was while he says all of this he's thinking about more than just how it was easier for the Lunarium to find him than the other way around. Him being bitter would be about him reflecting on not joining them earlier, stuff like that, but as we don't go and peer into characters' thoughts it couldn't be stated.

If that can't really translate without it implying the bitterness is about the fact of him being found by the Lunarium before he could find them, I'd rather just cut the sentence at "He smirks lightly" as self-deprecation doesn't seem appropriate to me either, as that's not what he's feeling.

File: data/coalition/lunarium intro.txt (hunk 2 above)
Comment (Arachi-Lover) [reply data/coalition/lunarium intro.txt:1632]: Maybe `... shall be those, who in our time of need do help us."` ? Or `..., who do help us in our time of need."` ?

File: data/coalition/lunarium intro.txt (hunk 4 above)
Comment (Arachi-Lover) [reply data/coalition/lunarium intro.txt:2018]: I see. The original change is fine then I think. Elirom's more thinking aloud to himself, rather than explaining out loud what he's trying to remember, so I think the original change fits better.

**Review by Arachi-Lover — APPROVED** [review]

File: data/coalition/lunarium intro.txt (hunk 1 above)
Comment (Arachi-Lover) [reply data/coalition/lunarium intro.txt:1301]: Yeah, just cutting it at "lightly" seems best.

**Review by Arachi-Lover — APPROVED** [review]
Looks all good now, thanks!
